src/containers/Network/UpgradeStatus.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import axios from 'axios'
+import BarChartVersion from './BarChartVersion'
+import NetworkTabs from './NetworkTabs'
+import Streams from '../shared/components/Streams'
+import Hexagons from './Hexagons'
+import { localizeNumber } from '../shared/utils'
+import { useLanguage } from '../shared/hooks'
+import Log from '../shared/log'
+
+export const UpgradeStatus = () => {
+  const [vList, setVList] = useState<any>([{}])
+  const [validations, setValidations] = useState([])
+  const [unlCount, setUnlCount] = useState(0)
+  const [stableVersion, setStableVersion] = useState<string | null>(null)
+  const { t } = useTranslation()
+  const language = useLanguage()
+
+  const aggregateData = (v: any[]) => {
+    if (!v) {
+      return []
+    }
+    let total = 0
+    const tempData: any[] = []
+    v.reduce((res, val) => {
+      if (!res[val.server_version]) {
+        res[val.server_version] = {
+          server_version: val.server_version,
+          count: 0,
+        }
+        tempData.push(res[val.server_version])
+      }
+      res[val.server_version].count += 1
+      total += 1
+      return res
+    }, {})
+
+    return tempData
+      .map((item) => ({
+        label: item.server_version ? item.server_version : ' NA ',
+        value: (item.count * 100) / total,
+        count: item.count,
+      }))
+      .sort((a, b) => (a.label > b.label ? 1 : -1))
+  }
+
+  useEffect(() => {
+    fetchStableVersion()
+    fetchData()
+    const interval = setInterval(fetchData, 5000)
```

Comment:
Should we really be fetching data so often? - If there's not very much downside to the request it's fine, but I'm not sure how costly the look up is. 

@ckniffen any thoughts on what a reasonable interval for this would be? (I would imagine that validators don't update their version very often, but if you just updated you may want to see that number update. Maybe just checking once when you load the page would be enough - and they have to refresh to see more?)